pkg/ui/app/containers/clusterViz.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+import {runVisualization, viewWidth, viewHeight} from "../js/sim/index";
+
+/**
+ * Renders the layout of the ClusterViz page. "ClusterViz" is an experimental
+ * visual map of the cluster, expressed as a directive 
```

Comment:
Trailed off in the middle of a thought; I was probably trying to write "directed graph", decided that it wasn't a directed graph, left to figure out what it actually was, then never came back to this comment.